p2plending/frontend/src/components/LanguageItem.js

```diff
@@ -13,6 +13,7 @@ class LanguageItem extends Component {
   }
   
   render() {
+    console.log(this.props)
```

Comment:
can you remove this console.log? It's generally bad practice to have debugging console logs pushed.

Reply:
OK - will do and send new commit and be more carefull going forward